### ISSUE TITLE
 this series line keeps disappearing and re appeaaring when an event is applied to it.

### ISSUE THREAD

**Issue by collinstinegaCodewexy** [amcharts/amcharts5#553]
I am using a  live chart demo provided by the documentation of amCharts5. I what the data to keep coming in but the xAxis should not be moving with it. I have achieved this by declaring the min and max values in the xAxis settings.
```JavaScript
let xAxis = chart.xAxes.push(
    am5xy.DateAxis.new(root, {
      maxDeviation: 1,
      groupData: false,
      wheelX: "panX",
      wheelY: "zoomX",
      min: data[0].date,
      max: data[data.length - 1].date,
      extraMax: 3.8,
      extraMin: -0.5,
      baseInterval: {
        timeUnit: "second",
        count: 1,
      },
      renderer: am5xy.AxisRendererX.new(root, {
        minGridDistance: 50,
      }),
      tooltip: am5.Tooltip.new(root, {}),
    })
);
````

I've also removed the function that deletes data from the array so I can have a continuous line  
`series.data.removeIndex(0);`
 the line keeps disappearing and reappearing when the graph is stretched or an event is applied to is 
here is the demo. I am use vue.js  but the logic is the same 
https://codesandbox.io/s/goofy-brahmagupta-ozzoob?file=/src/App.vue

**Comment by martynasma** [comment]
I'm trying to notice the disappearance/appearance effect you mentioned. I can't quite capture it.

I mean data items are being added, and the min/max is being updated. The chart updates fine for me.

Here's a pure JS version:
https://codepen.io/team/amcharts/pen/poLqdgK/5385253dd4e75b691aa75b40bd2442d6?editors=0010

What am I missing?

**Comment by collinstinegaCodewexy** [comment]
after 3 seconds the line disappears and reappears when the chart is stretched or tabs are switched. check your pen I can still see the error there I can see it on your pen. when the page has loaded, don't touch anything. wait for 2-4 seconds

**Comment by martynasma** [comment]
I'm sorry I don't see it. Which browser are you testing with?

**Comment by collinstinegaCodewexy** [comment]
chrome

**Comment by collinstinegaCodewexy** [comment]
which browser are you on ?

**Comment by martynasma** [comment]
Chrome, too.

I've now had other members review this. Nobody seems to notice anything disappearing/appearing, unless we're missing something.

**Comment by collinstinegaCodewexy** [comment]
try this 
https://codepen.io/tinega/pen/GRxzgOB
this is how the chart will be presented

**Comment by martynasma** [comment]
I don't get it. The chart stops updating after a few seconds until something is done on it.

Is the same happening to you, too?

**Comment by collinstinegaCodewexy** [comment]
yeah, how can I fix that?
is it an internal bug

**Comment by collinstinegaCodewexy** [comment]
this is urgent, anyone who can fix it ?

**Comment by collinstinegaCodewexy** [comment]
this is urgent any one who can fix it?

**Comment by martynasma** [comment]
This seems related to min/max and extraMin/extraMax combination in X axis settings. We're looking into it.

**Comment by martynasma** [comment]
This issue was fixed in dev version, and will be released soon.

We'll notify you here when that happens.

**Comment by collinstinegaCodewexy** [comment]
Do you have a date for the next release

**Comment by martynasma** [comment]
Most probably this week. No specific day.

**Comment by martynasma** [comment]
Fixed in [5.2.18](https://github.com/amcharts/amcharts5/releases/tag/5.2.18).

## [5.2.18] - 2022-08-24

### Added
- `disabled` key support added to `DropdownList` item interface.

### Fixed
- Removed leftover debug code in `IndicatorControl`.
- `selected` event in `IndicatorControl` was not firing.
- Adding an adapter on Y-axis' labels could cause flickers on the chart.
- `maxWidth`/`maxHeight` of a `Container` was ignored when arranging objects by Horizontal or Vertical layout.
- `maxWidth`/`maxHeight` of a `Container` was ignored when drawing background.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
